joomla/joomla-cms
# [4.0] Remove email-confirm from registration.

### Summary of Changes

This PR removes the email confirmation field from the registration form, as discussed at the sprint.

### Testing Instructions

- Ensure the registration form works as expected.

@wilsonge @laoneo could you just check to see if I've missed anything out?
